src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class Ball extends Lottery {

```

Comment:
```suggestion
public class Ball {

```

class Ball should not extends Lottery (as they don't have common state)